nova/nova/models.py

```diff
@@ -32,45 +31,63 @@ class User(AbstractUser):
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = ['created_at', 'date_of_birth', 'groups', 'first_name', 'last_name']
 
-
 class Article(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
-    # foreign key -> many to one
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    ## foreign key -> many to one
+    author = models.ForeignKey('User', on_delete=models.CASCADE, default=0)
 
     title = models.CharField(max_length=100)
 
     content = models.CharField(max_length=3000)
 
     rating = models.FloatField(blank=True, default=0)
 
-    REQUIRED_FIELDS = ['created_at', 'title', 'author', 'content']
+    REQUIRED_FIELDS = ['created_at', 'title',  'content']
+
 
 
 class TradingEquipment(models.Model):
     type = models.CharField(max_length=30)
 
     name = models.CharField(max_length=50)
 
-    daily_prices = ArrayField(ArrayField(models.FloatField()))
+    daily_prices =  ArrayField(models.FloatField(), default=list)
 
-    current_price = models.FloatField()
+    current_price = models.FloatField(default=0)
```

Comment:
I don't think we need a field for the current price. It should be the last element of the array daily_prices

Reply:
I realized that too. Currently I'm working on a different type of relation between equipments, parities and the time-dependent data.